python_modules/libraries/lakehouse/lakehouse/house.py

```diff
@@ -95,10 +97,166 @@ def __init__(
         self._assets = QueryableAssetSet(assets or [])
 
     def build_pipeline_definition(self, name, assets_to_update):
+        solid_defs, solid_deps = self._get_solid_deps_and_defs(assets_to_update)
+
+        return PipelineDefinition(
+            name=name,
+            solid_defs=list(solid_defs.values()),
+            mode_defs=self._mode_defs,
+            dependencies=solid_deps,
+            preset_defs=self._preset_defs,
+        )
+
+    def build_composite_solid_definition(self, name, assets_to_update, include_nothing_input=False):
+        """Build a composite solid definition for the assets in `assets_to_update`.
+
+        By default the composite solid will not accept any inputs. If you need to run this composite
+        _after_ other solids have run, pass `include_nothing_input=True`, which will add a single
+        input of type `Nothing` to the solid.
+
+        Examples:
+
+            .. code-block:: python
+
+            from pyspark.sql import DataFrame, Row
```

Comment:
This is a pretty heavy example - would it be possible to communicate the same usage with fewer elements?

Reply:
I reckon so, if the examples don't need to be runnable - in that case we could just have the solids and pipelines, and remove all the rest.